apps/docs/security/github-scanning.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: "GitHub Secret Scanning"
+description: "Learn about"
+---
+
+Unkey has partnered with GitHub to scan repositories for leaked keys. 
+
+`GitHub Secret Scanning` uses a regular expression to scan repositories for keys matching **Unkey root keys**. 
+If a key is found Github will send a request to our API. Once the key has been validated to be a real key, we will email you with details about this leak. 
```

Comment:
Calling our API is an implementation detail the end user won't care about, it's enough to say that github notifies us, we verify the claim and then notify the user if necessary